### ISSUE TITLE
[Web][Accessibility] Actions with the OpenUrl onAction should render as an a tag

### ISSUE THREAD

**Issue by felixvanleeuwen** [microsoft/AdaptiveCards#4947]
**What platform is your feature request related to? (Delete other platforms).**

* .NET HTML
* .NET WPF
* Android
* iOS
* JavaScript
* UWP

**Is your feature request related to a problem? Please describe.**
[A European law has been passed that forces public websites to adhere to a AA accessibility rating.](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/02.01.02_60/en_301549v020102p.pdf)
Using adaptive cards prevents this.
All actions are rendered the same, as buttons.
For blind or poor sighted people who browse the web (using screen readers) this is incredibly confusing.

**Describe the solution you'd like**
By default, the `Action.OpenUrl` is rendered as an `<a>`-tag.
Users who rely on sight don't notice a difference.

**Describe alternatives you've considered**
Some rendering middleware, but could not find one that exists.

**Comment by dclaux** [comment]
@felixvanleeuwen since version 2.3 of the JS SDK, all OpenUrl actions are rendered with the `"aria-role": "link"` attribute, which should be enough to satisfy the regulation. Rendering them as an `<a>` tag is only necessary if the proper aria-role is missing, which isn't the case.

**Comment by dclaux** [comment]
Scrap what I wrote above. While that _was_ true, another accessibility change has been applied on top of it that changes buttons' role to "menuitem".

We're going to have to look into this and determine what the right solution is.

**Comment by felixvanleeuwen** [comment]
Any news on this?

**Comment by paulcam206** [comment]
This issue has been fixed in #6763
